jwst/datamodels/schemas/guider_cds.schema.yaml

```diff
@@ -0,0 +1,132 @@
+allOf:
```

Comment:
I don't see the "guider_cds.schema.yaml" file used in any of the data models. Only guider_fgs_raw.schema.yaml and guider_fgs_cal.schema.yaml.